src/main/java/org/fhirvp/websocket/FHIRNotificationClientEndpoint.java

```diff
@@ -0,0 +1,27 @@
+package org.fhirvp.websocket;
+
+import io.quarkus.logging.Log;
+import org.fhirvp.notification.NotificationHandler;
+
+import javax.enterprise.inject.spi.CDI;
+import javax.websocket.ClientEndpoint;
+import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
+
+@ClientEndpoint
+public class FHIRNotificationClientEndpoint {
+
+    @OnOpen
+    void onOpen(Session session) {
+        Log.info("WebSocket session with URI '" + session.getRequestURI() + "' opened.");
+    }
+
+    @OnMessage
+    void handleIncomingFHIRNotification(String msg) {
+        Log.info("Received FHIR Notification: " + msg);
```

Comment:
Please add a `TODO:` here to remove this, logging FHIR data can be problematic if it contains personally identifying information.

Reply:
Good catch! I will remove all sensitive data from the log.

Reply:
Addressed at 54ec843